apps/server/src/modules/class/service/class.service.spec.ts

```diff
@@ -202,7 +202,14 @@ describe(ClassService.name, () => {
 				expect(classesRepo.findAllByUserId).toBeCalledWith(userId1.toHexString());
 			});
 
-			it('should update classes without updated user', async () => {
+			it('should call classesRepo.deleteUser', async () => {
+				const { userId1 } = setup();
+				await service.deleteUserData(userId1.toHexString());
```

Comment:
Why not move the `toHexString()` into `setup()` and just deal with `EntityId` values?